src/popup/utils/utils.ts

```diff
@@ -442,10 +446,18 @@ export function getAccountNameToDisplay(acc: IAccount | undefined) {
 }
 
 export function defaultTransactionSortingCallback(
-  a: ITransaction,
-  b: ITransaction,
+  a: ITransaction | IActiveMultisigTx,
+  b: ITransaction | IActiveMultisigTx,
 ) {
-  const [aMicroTime, bMicroTime] = [a, b].map((tr) => (new Date(tr.microTime)).getTime());
+  const [aMicroTime, bMicroTime] = [a, b].map(
+    (transaction) => (new Date(
+      (transaction as any).isMultisigTransaction
+        ? (transaction as IActiveMultisigTx).createdAt
```

Comment:
As far as I know `createdAt` refers to the Generalised Account, not the transaction.

Reply:
There are a `createdAt` in the response from `https://ga-multisig-backend-mainnet.prd.aepps.com/tx/...`, which is referring to the creation time of a transaction proposal.

Reply:
But in this case the `createdAt` is related to an activeAccount
https://github.com/aeternity/superhero-wallet/blob/247e8e08368c8afca28e1dd48c124bfc2c6a9222/src/composables/multisigTransactions.ts#L78-L84